.github/actions/check_cirrus_cron/cron_failures.sh

```diff
@@ -102,6 +97,16 @@ fi
 #     }
 #   }
 # }
+
+# This should never ever return an empty-list, unless there are no cirrus-cron
+# jobs defined for the repository.  In that case, this monitoring script shouldn't
+# be running anyway.
+if ! jq -e '.data.ownerRepository.cronSettings' ./artifacts/reply.json &> /dev/null
```

Comment:
Suggestion:
```bash
foo='.data.ownerRepository.cronSettings'
if ! jq -e $foo ...
    err "GraphQL reply did not contain $foo"
```

Reply:
Yeah that line is getting a bit close to needing a scroll-bar, and that error would be more clear.  Okay I'll fix...